smart-contract/src/contract.rs

```diff
@@ -1484,10 +1648,31 @@ mod tests {
         // Assume the customer has a balance of tokens + the required attribute.
         let dcc = coin(1000, "dcc.coin");
         deps.querier
-            .base
-            .update_balance(Addr::unchecked("customer"), vec![dcc]);
-        deps.querier
-            .with_attributes("customer", &[("bank.kyc.pb", "ok", "string")]);
+            .mock_querier
+            .update_balance("customer", vec![dcc]);
+
+        // TODO - fix test since mock response returns same result no matter the input
```

Comment:
waiting on resolution from provwasm team about mocking not considering the inputs of the query